e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/check-details.spec.js

```diff
@@ -245,15 +249,13 @@ context('Deal cancellation - check details', () => {
         });
       });
     });
-  });
 
-  describe('when logged in as a non-PIM user', () => {
-    beforeEach(() => {
-      cy.login(T1_USER_1);
-    });
+    describe('"delete deals" button', () => {
```

Comment:
This describe block is needed to ensure this test executes after the rest (clicking the button cannot be undone). 

Annoyingly if the `it` existed outside of this `describe` block it is hoisted above the describes & the tests fail

Reply:
E.g, even when the tests are written in the following order
![image](https://github.com/user-attachments/assets/77349ec0-e732-4bb4-9f5e-205af2d464e9)

They execute in this order
![image](https://github.com/user-attachments/assets/92584861-02d8-47e2-8d84-0d8f9b92ae88)

and the `reason for cancelling - change link` tests fail as the deal cant be cancelled again